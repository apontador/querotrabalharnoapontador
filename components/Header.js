import styled from 'styled-components';

const Header = styled.header`
 #logo {
     float: left;
     margin-left: 16px;
     display: flex;
     align-items: center;
     height: 38px;
 }
 #menu-trigger {
     padding: 3px;
     margin: 10px 0;
     cursor: pointer;
     float: left;
 }
 .component#login {
     margin-left: 16px;
 }
 #header-search {
     width: 614px;
     margin: 0 auto;
 }
 .with-search form#search {
     width: 100%;
 }
 form#search {
     display: flex;
     justify-content: center;
     flex-direction: column;
     position: relative;
     z-index: 9999;
 }
 form#search .component {
     border: 1px solid #ff8207;
     box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.24), 0 0 2px 0 rgba(0, 0, 0, 0.12);
     border-radius: 2px;
     margin: 0 auto;
     display: flex;
 }
 form#search .component .box {
     width: 336px;
 }
 form#search .component .box#location {
     width: 222px;
 }
 form#search input {
     padding: 16px 8.8px 15px;
     width: 100%;
     border-color: transparent;
     outline: 0;
     font-size: 1.4em;
     -webkit-appearance: none;
     outline: 0;
 }
 form#search .component .box#search-place {
     width: 50px;
 }
 form#search button {
     border-color: transparent;
     width: 100%;
     background: #ff8207 url(https://newpoi-assets.apontador.com.br/website/images/buscar.svg) no-repeat;
     background-position: center left 15px;
     font-size: 1.6em;
     text-transform: none;
     color: #fff;
     padding: 15px 5px 15px 25px;
     height: 100%;
 }
 form#search button {
     text-indent: -9999px;
 }
 button {
     color: #9012fe;
     border-color: #9012fe;
 }
 #login button {
     margin-right: 0;
     border: 1px solid transparent;
     color: #ff8207;
     background-color: #eee;
 }
 @media screen and (max-width: 500px) {
     form#search button {
         background: #ff8207 url(https://newpoi-assets.apontador.com.br/website/images/buscar.svg) no-repeat;
         background-position: center;
         background-size: 20px;
     }
 }
 @media screen and (max-width: 530px) {
     .component#login button {
         border: 0;
         padding: 0;
         background: url(https://newpoi-assets.apontador.com.br/website/images/entrar_mobile.svg) no-repeat;
         text-indent: -9999px;
         width: 20px;
         height: 20px;
     }
     .component#login {
         margin: 10px 0;
     }
 }
 @media screen and (max-width: 600px) {
     form#search .component .box#query {
         width: 80%;
     }
 }
 @media screen and (max-width: 716px) {
     form#search {
         display: block;
     }
     form#search .component {
         width: 100%;
         background: #fff;
         display: flex;
         justify-content: space-between;
     }
     form#search .component .box#query {
         width: 90%;
     }
     form#search .component .box#location {
         display: none;
     }
     form#search .component .box#search-place {
         float: right;
     }
     form#search button {
         text-indent: -9999px;
         padding: 0;
         height: 51px;
     }
 }
 @media screen and (max-width: 1150px) {
     .component#anuncie {
         display: none;
     }
 }
 @media screen and (max-width: 960px) {
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
     #logo {
         margin: 0 auto;
     }
     .box.right {
         position: absolute;
         right: 16px;
         top: 16px;
     }
     .box.left {
         order: 1;
         flex-grow: 2;
         margin-right: 0;
         justify-content: center;
         display: flex;
     }
     #header-search {
         order: 4;
         width: 100%;
         flex-grow: 2;
         margin-top: 20px;
     }
     #menu-trigger {
         position: absolute;
         left: 16px;
         top: 16px;
     }
 }
`;


export default () => (
  <Header className="with-search">
    <div className="box left">
      <div id="menu-trigger"><img alt="Menu" src="https://newpoi-assets.apontador.com.br/website/images/menu.svg" title="Menu"/></div>
      <div id="logo">
        <a href="https://www.apontador.com.br/"><img src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"/></a>
      </div>
    </div>
    <div id="header-search">
      <form action="https://www.apontador.com.br/local/search.html" className="row" id="search" name="search">
        <div className="component">
          <div className="box" id="query">
            <input autocomplete="off" id="query_field" name="q" placeholder="O que você procura?" type="text" value="" />
          </div>
          <div className="box" id="location">
            <input autocomplete="off" id="location_field" name="loc" placeholder="Cidade, Estado" type="text" value="" />
          </div>
          <div className="box" id="search-place">
            <button type="submit">Buscar</button>
          </div>
        </div>
      </form>
    </div>
    <div className="box right">
      <div className="component" id="anuncie">
        <a href="https://www.apontador.com.br/mais/planos/" target="_blank"><button id="anuncie">Anuncie seu Negócio</button></a>
      </div>
      <div className="component" id="login">
        <button id="login-trigger">Entrar</button>
      </div>
    </div>
  </Header>
)