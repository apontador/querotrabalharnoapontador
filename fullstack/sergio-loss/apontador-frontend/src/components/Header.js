import React, { Fragment } from 'react'
import { connect } from 'react-redux'
import { Link, Redirect } from 'react-router-dom'

import { AppBar, Toolbar, Grid, Button } from '@material-ui/core'

import logo from '../images/logo-apontador.png'
import search from '../images/search.png'

import { LoadStateResultSearch, LoadStateSearchText } from '../actions/PlaceAction'
import { LoadStateRedirect, ResetRedirect } from '../actions/RedirectAction'

import { doSearch } from '../tasks/PlaceTask'

const Header = ({LoadStateResultSearch, redirect, LoadStateRedirect, ResetRedirect, LoadStateSearchText, search_text, doSearch}) => {

    //css
    const style = {
        header: {
            backgroundColor: '#ffffff',
            color: '#000000',
            paddingTop: '10px',
            paddingBottom: '10px'
        },
        grow: {
            flexGrow: '1'
        },
        btnSearch: {
            height: '45px',
            width: '45px',
            backgroundColor: '#FF8207',
            border: 0,
            borderTopRightRadius: '0.2em',
            borderBottomRightRadius: '0.2em',
            cursor: 'pointer',
            marginLeft: '-1px'
        },
        txtSearch: {
            border: '1px solid #FF8207',
            fontSize: 'medium',
            borderTopLeftRadius: '0.2em',
            borderBottomLeftRadius: '0.2em',
            paddingTop: '12px',
            paddingBottom: '10px',
            paddingLeft: '10px',
            width: '100%',
            maxWidth: '650px'
        },
        btnNewPlace:{
            color: '#693EBC',
            border: '1px solid #693EBC'            
        }
    }

    const handleSearchClick = () => {
        if(search_text)
            doSearch(search_text, 1, result_search=>{
                LoadStateRedirect({ to: "/" })
                LoadStateResultSearch(result_search)
                ResetRedirect()
            })
    }

    return (
        <Fragment>
            <AppBar style={style.header} position="fixed">
                <Toolbar >
                    <Grid container spacing={1} alignItems="center">                    
                        <Grid item xs={12} sm={3}>
                            <Link to="/" className="no-text-decoration">
                                <img src={logo} alt="Apontador"/>
                            </Link>
                        </Grid>                    
                        <Grid item xs={12} sm={6}>
                            <div style={{display:'flex', justifyContent:'center'}}>
                                <input type="text" style={style.txtSearch} className="no-border-on-focus" onChange={e=>LoadStateSearchText(e.target.value)} value={search_text} />
                                <button style={style.btnSearch} className="no-border-on-focus" onClick={handleSearchClick}>
                                    <img src={search} alt="Search"/>
                                </button>                                
                            </div>                       
                        </Grid>
                        <Grid item xs={12} sm={3}>
                            <Grid container justify="flex-end">
                                <Link to="/new-place" className="no-text-decoration">
                                    <Button variant="outlined" style={style.btnNewPlace}>Cadastre um local</Button>
                                </Link>                            
                            </Grid>                        
                        </Grid>
                    </Grid>
                </Toolbar>
            </AppBar>
            { redirect.to ? <Redirect to={redirect.to} /> : "" } 
        </Fragment>
    )
}

const mapStateToProps = state => ({
    redirect: state.RedirectReducer.redirect,
    search_text: state.PlaceReducer.search_text
});
  
const mapDispatchToProps = dispatch => {
    return {
        LoadStateResultSearch: result_search => dispatch(LoadStateResultSearch(result_search)),
        LoadStateRedirect: redirect => dispatch(LoadStateRedirect(redirect)),
        ResetRedirect: () => dispatch(ResetRedirect()),
        LoadStateSearchText: search_text => dispatch(LoadStateSearchText(search_text)),
        doSearch
    };
};

export default connect(
    mapStateToProps
    ,mapDispatchToProps
)(Header);