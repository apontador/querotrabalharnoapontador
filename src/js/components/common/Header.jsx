'use strict';

import React from 'react';
import { Nav, Navbar, NavDropdown, NavItem, MenuItem, FormControl } from 'react-bootstrap';

const Header = React.createClass({
    render() {
        return (
            <header>
                <Navbar fixedTop className="apontador-header">
                    <Navbar.Header>
                        <Navbar.Brand>
                            <a href="#">
                                <img className="img-responsive" alt="Apontador" src="https://clientmanager.apontador-assets.com/prod/base/images/newheader/logo.png?v=131"/>                            
                            </a>
                        </Navbar.Brand>
                        <Navbar.Toggle />
                        </Navbar.Header>
                    <Navbar.Collapse>
                        <Navbar.Form pullLeft>
                            <FormControl type="search" placeholder="Search" />
                        </Navbar.Form>
                        <Nav>
                            <NavDropdown eventKey={3} title="Categorias" id="dropdown-categorias">
                                <MenuItem eventKey={3.1}>
                                    <span className="glyphicon glyphicon-cutlery"></span>
                                    {' '}
                                    Restaurantes
                                </MenuItem>
                                <MenuItem eventKey={3.2}>
                                    <span className="glyphicon glyphicon-glass"></span>
                                    {' '}
                                    Bares
                                </MenuItem>
                                <MenuItem eventKey={3.3}>
                                    <span className="glyphicon glyphicon-film"></span>
                                    {' '}
                                    Cinemas
                                </MenuItem>
                                <MenuItem eventKey={3.4}>
                                    <span className="glyphicon glyphicon-heart-empty"></span>
                                    {' '}
                                    Teatros
                                </MenuItem>
                                <MenuItem divider />
                                <MenuItem eventKey={3.5}>Todas Categorias</MenuItem>
                            </NavDropdown>
                        </Nav>
                        <Nav pullRight>
                            <NavItem eventKey={2} href="#">
                                <span className="glyphicon glyphicon-user"></span>
                                {' '}
                                Entrar
                            </NavItem> 
                            <NavItem eventKey={1} href="#">
                                <span className="glyphicon glyphicon-envelope"></span>
                                {' '}
                                Anuncie seu neg√≥cio
                            </NavItem>
                            <NavDropdown eventKey={4} title="Portugues" id="dropdown-lang">
                                <MenuItem eventKey={4.1}>English</MenuItem>
                                <MenuItem divider />
                                <MenuItem eventKey={4.2}>Spanish</MenuItem>
                            </NavDropdown>
                        </Nav>
                    </Navbar.Collapse>
                </Navbar>
            </header>
        );
    }
});

export default Header;
