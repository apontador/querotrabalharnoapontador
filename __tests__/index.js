import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Header from 'components/Header';

configure({ adapter: new Adapter() });

describe('Header', () => {
    it('should be defined', () => {
        expect(Header).toBeDefined();
    });

    it('should render correctly', () => {
        const tree = shallow(
            <Header />
        );
        expect(tree).toMatchSnapshot();
    });

});