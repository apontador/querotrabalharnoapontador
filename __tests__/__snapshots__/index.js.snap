// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <div
          className="box left"
        >
          <div
            id="menu-trigger"
          >
            <img
              alt="Menu"
              src="https://newpoi-assets.apontador.com.br/website/images/menu.svg"
              title="Menu"
            />
          </div>
          <div
            id="logo"
          >
            <a
              href="https://www.apontador.com.br/"
            >
              <img
                src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
              />
            </a>
          </div>
        </div>,
        <div
          id="header-search"
        >
          <form
            action="https://www.apontador.com.br/local/search.html"
            className="row"
            id="search"
            name="search"
          >
            <div
              className="component"
            >
              <div
                className="box"
                id="query"
              >
                <input
                  autocomplete="off"
                  id="query_field"
                  name="q"
                  placeholder="O que você procura?"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="box"
                id="location"
              >
                <input
                  autocomplete="off"
                  id="location_field"
                  name="loc"
                  placeholder="Cidade, Estado"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="box"
                id="search-place"
              >
                <button
                  type="submit"
                >
                  Buscar
                </button>
              </div>
            </div>
          </form>
        </div>,
        <div
          className="box right"
        >
          <div
            className="component"
            id="anuncie"
          >
            <a
              href="https://www.apontador.com.br/mais/planos/"
              target="_blank"
            >
              <button
                id="anuncie"
              >
                Anuncie seu Negócio
              </button>
            </a>
          </div>
          <div
            className="component"
            id="login"
          >
            <button
              id="login-trigger"
            >
              Entrar
            </button>
          </div>
        </div>,
      ],
      "className": "with-search",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              id="menu-trigger"
            >
              <img
                alt="Menu"
                src="https://newpoi-assets.apontador.com.br/website/images/menu.svg"
                title="Menu"
              />
            </div>,
            <div
              id="logo"
            >
              <a
                href="https://www.apontador.com.br/"
              >
                <img
                  src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                />
              </a>
            </div>,
          ],
          "className": "box left",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Menu"
                src="https://newpoi-assets.apontador.com.br/website/images/menu.svg"
                title="Menu"
              />,
              "id": "menu-trigger",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "alt": "Menu",
                "src": "https://newpoi-assets.apontador.com.br/website/images/menu.svg",
                "title": "Menu",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <a
                href="https://www.apontador.com.br/"
              >
                <img
                  src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                />
              </a>,
              "id": "logo",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                />,
                "href": "https://www.apontador.com.br/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "src": "https://newpoi-assets.apontador.com.br/website/images/logo_small.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "a",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": <form
            action="https://www.apontador.com.br/local/search.html"
            className="row"
            id="search"
            name="search"
          >
            <div
              className="component"
            >
              <div
                className="box"
                id="query"
              >
                <input
                  autocomplete="off"
                  id="query_field"
                  name="q"
                  placeholder="O que você procura?"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="box"
                id="location"
              >
                <input
                  autocomplete="off"
                  id="location_field"
                  name="loc"
                  placeholder="Cidade, Estado"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="box"
                id="search-place"
              >
                <button
                  type="submit"
                >
                  Buscar
                </button>
              </div>
            </div>
          </form>,
          "id": "header-search",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "action": "https://www.apontador.com.br/local/search.html",
            "children": <div
              className="component"
            >
              <div
                className="box"
                id="query"
              >
                <input
                  autocomplete="off"
                  id="query_field"
                  name="q"
                  placeholder="O que você procura?"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="box"
                id="location"
              >
                <input
                  autocomplete="off"
                  id="location_field"
                  name="loc"
                  placeholder="Cidade, Estado"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="box"
                id="search-place"
              >
                <button
                  type="submit"
                >
                  Buscar
                </button>
              </div>
            </div>,
            "className": "row",
            "id": "search",
            "name": "search",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="box"
                  id="query"
                >
                  <input
                    autocomplete="off"
                    id="query_field"
                    name="q"
                    placeholder="O que você procura?"
                    type="text"
                    value=""
                  />
                </div>,
                <div
                  className="box"
                  id="location"
                >
                  <input
                    autocomplete="off"
                    id="location_field"
                    name="loc"
                    placeholder="Cidade, Estado"
                    type="text"
                    value=""
                  />
                </div>,
                <div
                  className="box"
                  id="search-place"
                >
                  <button
                    type="submit"
                  >
                    Buscar
                  </button>
                </div>,
              ],
              "className": "component",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": <input
                    autocomplete="off"
                    id="query_field"
                    name="q"
                    placeholder="O que você procura?"
                    type="text"
                    value=""
                  />,
                  "className": "box",
                  "id": "query",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "autocomplete": "off",
                    "id": "query_field",
                    "name": "q",
                    "placeholder": "O que você procura?",
                    "type": "text",
                    "value": "",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "input",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": <input
                    autocomplete="off"
                    id="location_field"
                    name="loc"
                    placeholder="Cidade, Estado"
                    type="text"
                    value=""
                  />,
                  "className": "box",
                  "id": "location",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "autocomplete": "off",
                    "id": "location_field",
                    "name": "loc",
                    "placeholder": "Cidade, Estado",
                    "type": "text",
                    "value": "",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "input",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": <button
                    type="submit"
                  >
                    Buscar
                  </button>,
                  "className": "box",
                  "id": "search-place",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Buscar",
                    "type": "submit",
                  },
                  "ref": null,
                  "rendered": "Buscar",
                  "type": "button",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "form",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="component"
              id="anuncie"
            >
              <a
                href="https://www.apontador.com.br/mais/planos/"
                target="_blank"
              >
                <button
                  id="anuncie"
                >
                  Anuncie seu Negócio
                </button>
              </a>
            </div>,
            <div
              className="component"
              id="login"
            >
              <button
                id="login-trigger"
              >
                Entrar
              </button>
            </div>,
          ],
          "className": "box right",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <a
                href="https://www.apontador.com.br/mais/planos/"
                target="_blank"
              >
                <button
                  id="anuncie"
                >
                  Anuncie seu Negócio
                </button>
              </a>,
              "className": "component",
              "id": "anuncie",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <button
                  id="anuncie"
                >
                  Anuncie seu Negócio
                </button>,
                "href": "https://www.apontador.com.br/mais/planos/",
                "target": "_blank",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Anuncie seu Negócio",
                  "id": "anuncie",
                },
                "ref": null,
                "rendered": "Anuncie seu Negócio",
                "type": "button",
              },
              "type": "a",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <button
                id="login-trigger"
              >
                Entrar
              </button>,
              "className": "component",
              "id": "login",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Entrar",
                "id": "login-trigger",
              },
              "ref": null,
              "rendered": "Entrar",
              "type": "button",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <div
            className="box left"
          >
            <div
              id="menu-trigger"
            >
              <img
                alt="Menu"
                src="https://newpoi-assets.apontador.com.br/website/images/menu.svg"
                title="Menu"
              />
            </div>
            <div
              id="logo"
            >
              <a
                href="https://www.apontador.com.br/"
              >
                <img
                  src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                />
              </a>
            </div>
          </div>,
          <div
            id="header-search"
          >
            <form
              action="https://www.apontador.com.br/local/search.html"
              className="row"
              id="search"
              name="search"
            >
              <div
                className="component"
              >
                <div
                  className="box"
                  id="query"
                >
                  <input
                    autocomplete="off"
                    id="query_field"
                    name="q"
                    placeholder="O que você procura?"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  className="box"
                  id="location"
                >
                  <input
                    autocomplete="off"
                    id="location_field"
                    name="loc"
                    placeholder="Cidade, Estado"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  className="box"
                  id="search-place"
                >
                  <button
                    type="submit"
                  >
                    Buscar
                  </button>
                </div>
              </div>
            </form>
          </div>,
          <div
            className="box right"
          >
            <div
              className="component"
              id="anuncie"
            >
              <a
                href="https://www.apontador.com.br/mais/planos/"
                target="_blank"
              >
                <button
                  id="anuncie"
                >
                  Anuncie seu Negócio
                </button>
              </a>
            </div>
            <div
              className="component"
              id="login"
            >
              <button
                id="login-trigger"
              >
                Entrar
              </button>
            </div>
          </div>,
        ],
        "className": "with-search",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                id="menu-trigger"
              >
                <img
                  alt="Menu"
                  src="https://newpoi-assets.apontador.com.br/website/images/menu.svg"
                  title="Menu"
                />
              </div>,
              <div
                id="logo"
              >
                <a
                  href="https://www.apontador.com.br/"
                >
                  <img
                    src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                  />
                </a>
              </div>,
            ],
            "className": "box left",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="Menu"
                  src="https://newpoi-assets.apontador.com.br/website/images/menu.svg"
                  title="Menu"
                />,
                "id": "menu-trigger",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "alt": "Menu",
                  "src": "https://newpoi-assets.apontador.com.br/website/images/menu.svg",
                  "title": "Menu",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://www.apontador.com.br/"
                >
                  <img
                    src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                  />
                </a>,
                "id": "logo",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    src="https://newpoi-assets.apontador.com.br/website/images/logo_small.svg"
                  />,
                  "href": "https://www.apontador.com.br/",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "src": "https://newpoi-assets.apontador.com.br/website/images/logo_small.svg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "a",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": <form
              action="https://www.apontador.com.br/local/search.html"
              className="row"
              id="search"
              name="search"
            >
              <div
                className="component"
              >
                <div
                  className="box"
                  id="query"
                >
                  <input
                    autocomplete="off"
                    id="query_field"
                    name="q"
                    placeholder="O que você procura?"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  className="box"
                  id="location"
                >
                  <input
                    autocomplete="off"
                    id="location_field"
                    name="loc"
                    placeholder="Cidade, Estado"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  className="box"
                  id="search-place"
                >
                  <button
                    type="submit"
                  >
                    Buscar
                  </button>
                </div>
              </div>
            </form>,
            "id": "header-search",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "action": "https://www.apontador.com.br/local/search.html",
              "children": <div
                className="component"
              >
                <div
                  className="box"
                  id="query"
                >
                  <input
                    autocomplete="off"
                    id="query_field"
                    name="q"
                    placeholder="O que você procura?"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  className="box"
                  id="location"
                >
                  <input
                    autocomplete="off"
                    id="location_field"
                    name="loc"
                    placeholder="Cidade, Estado"
                    type="text"
                    value=""
                  />
                </div>
                <div
                  className="box"
                  id="search-place"
                >
                  <button
                    type="submit"
                  >
                    Buscar
                  </button>
                </div>
              </div>,
              "className": "row",
              "id": "search",
              "name": "search",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="box"
                    id="query"
                  >
                    <input
                      autocomplete="off"
                      id="query_field"
                      name="q"
                      placeholder="O que você procura?"
                      type="text"
                      value=""
                    />
                  </div>,
                  <div
                    className="box"
                    id="location"
                  >
                    <input
                      autocomplete="off"
                      id="location_field"
                      name="loc"
                      placeholder="Cidade, Estado"
                      type="text"
                      value=""
                    />
                  </div>,
                  <div
                    className="box"
                    id="search-place"
                  >
                    <button
                      type="submit"
                    >
                      Buscar
                    </button>
                  </div>,
                ],
                "className": "component",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": <input
                      autocomplete="off"
                      id="query_field"
                      name="q"
                      placeholder="O que você procura?"
                      type="text"
                      value=""
                    />,
                    "className": "box",
                    "id": "query",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "autocomplete": "off",
                      "id": "query_field",
                      "name": "q",
                      "placeholder": "O que você procura?",
                      "type": "text",
                      "value": "",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "input",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": <input
                      autocomplete="off"
                      id="location_field"
                      name="loc"
                      placeholder="Cidade, Estado"
                      type="text"
                      value=""
                    />,
                    "className": "box",
                    "id": "location",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "autocomplete": "off",
                      "id": "location_field",
                      "name": "loc",
                      "placeholder": "Cidade, Estado",
                      "type": "text",
                      "value": "",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "input",
                  },
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": <button
                      type="submit"
                    >
                      Buscar
                    </button>,
                    "className": "box",
                    "id": "search-place",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": null,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Buscar",
                      "type": "submit",
                    },
                    "ref": null,
                    "rendered": "Buscar",
                    "type": "button",
                  },
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": "form",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="component"
                id="anuncie"
              >
                <a
                  href="https://www.apontador.com.br/mais/planos/"
                  target="_blank"
                >
                  <button
                    id="anuncie"
                  >
                    Anuncie seu Negócio
                  </button>
                </a>
              </div>,
              <div
                className="component"
                id="login"
              >
                <button
                  id="login-trigger"
                >
                  Entrar
                </button>
              </div>,
            ],
            "className": "box right",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="https://www.apontador.com.br/mais/planos/"
                  target="_blank"
                >
                  <button
                    id="anuncie"
                  >
                    Anuncie seu Negócio
                  </button>
                </a>,
                "className": "component",
                "id": "anuncie",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": <button
                    id="anuncie"
                  >
                    Anuncie seu Negócio
                  </button>,
                  "href": "https://www.apontador.com.br/mais/planos/",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Anuncie seu Negócio",
                    "id": "anuncie",
                  },
                  "ref": null,
                  "rendered": "Anuncie seu Negócio",
                  "type": "button",
                },
                "type": "a",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <button
                  id="login-trigger"
                >
                  Entrar
                </button>,
                "className": "component",
                "id": "login",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Entrar",
                  "id": "login-trigger",
                },
                "ref": null,
                "rendered": "Entrar",
                "type": "button",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
